VIDEO_4_SCRIPT.md

```diff
@@ -0,0 +1,47 @@
+# Video 4: AsyncAPI and OpenAPI
+
+Hello again! Welcome to a new video about AsyncAPI. This video will discuss what AsyncAPI and OpenAPI have in common and how they differ. 
+
+Ready or not, here we go! 
+
+## What is OpenAPI?
+
+OpenAPI is a standard for describing application programming interfaces (APIs). This specification defines an open, vendor-independent description format for API services. In particular, OpenAPI can describe, develop, test, and document REST-compliant APIs.
+
+The current OpenAPI specification grew out of the predecessor project, Swagger. The development company SmartBear subjected the existing Swagger specification to an open license, handing over both maintenance and development of the initiative to OpenAPI. In addition to SmartBear, members of the OpenAPI initiative include industry giants such as Google, IBM, and Microsoft. The Linux Foundation also supports this project. 
+
+## Similarities between AsyncAPI and OpenAPI
+
+AsyncAPI's ultimate goal is to work with EDA architectures as simply as with Rest APIs. The AsyncAPI specification in the asynchronous world is analogous to OpenAPI in the synchronous world. 
+
```

Comment:
For a beginner audience, the mention of rest API here might stir confusion considering the fact that we have been discussing only open API and async API. I think a little clarification should work.